package.json

```diff
@@ -1914,6 +2064,15 @@
           ],
           "scope": "machine"
         },
+        "cmake.cpackArgs": {
```

Comment:
Should there also be a setting for the `cpack` executable?

Reply:
Yes, I missed adding it in the package.json but it is in all other places.